app/models/movie.rb

```diff
@@ -5,4 +5,10 @@ class Movie < ApplicationRecord
   validates :title, length: { maximum: 255, too_long: '%<count>s characters is the maximum allowed' }
   validates :description, length: { maximum: 30_000, too_long: '%<count>s characters is the maximum allowed' }
   validates :duration, numericality: { only_integer: true, greater_than: 0 }
+
+  def self.today
+    includes(:seances)
+      .where(seances: { starts_at: DateTime.current.beginning_of_day...DateTime.current.end_of_day })
+      .order('seances.starts_at')
+  end
```

Comment:
this should be a scope
```suggestion
  scope :today -> { includes(seances).where(seances: { starts_at: DateTime.current.beginning_of_day...DateTime.current.end_of_day } ) }
```
This way you can chain the `today` scope with other queries to make one big query
And sence this could be used with other queries I would extract ordering out to the controller